pgmpy/inference/ExactInference.py

```diff
@@ -228,6 +230,45 @@ def induced_graph(self, elimination_order):
         >>> inference.induced_graph(['C', 'D', 'A', 'B', 'E'])
         <networkx.classes.graph.Graph at 0x7f34ac8c5160>
         """
+        eliminated_variables = set()
+        working_factors = {node: {factor for factor in self.factors[node]}
```

Comment:
@pratyakshs: Should not even be needing to create this huge structure. Should be okay with just the scopes of the factors.